portfolio/src/main/webapp/index.html

```diff
@@ -19,3 +19,35 @@ <h1>Cynthia's Website</h1>
     </div>
   </body>
 </html>
+
```

Comment:
Your main html file should have only HTML, one HEAD, and one BODY tag. If you have multiple scripts then you add multiple script tags. Then in your body you add all your html elements (divs, buttons, paragraphs). See example below. If you don't want to show all of them in the beginning, then you use visibility to show/hide elements. Please let me know if you have any questions.

```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title></title>
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" href="style_2.css">
    <script src="script.js"></script>
    <script src="script_1.js"></script>
    <script src="script_2.js"></script>
  </head>
  <body>
    <div id="content">
      <p>...</p>
     <button>Click me</button>
    </div>
  </body>
</html>
```